src/client/app/offers/offers.html

```diff
@@ -43,15 +43,27 @@
                 <div class="col-xs-9 col-sm-9 col-md-9 col-lg-9">
                     <div class="date">
                         <span>10/07/2015</span><br>
-                        <span class="activeNeed">Катя</span>
-                        <span class="Need">Єдинорог</span>
+                        </div class="needInfo">
+                        <div>
+                        <div class="activeNeed">Катя</div>
+                         <div class ="offerInfo" ng-hide="editorEnabled">
+                        {{offerText}}
+                        </div>
+                        <div ng-show="editorEnabled">
+                          <textarea class="form-control" rows="5" ng-model="editableOffer" ng-show="editorEnabled"></textarea>
+                          <a href="#" ng-click="save()">Зберегти?</a>
```

Comment:
Maybe a button instead of a link? :) if you click on this, you might be redirected to another page.